Correct Ubuntu distributions and signature

valdearg opened the pull request:
Corrects the error with the Ubuntu distribution names being incorrect. 
Simplifies the command to add repository. 
Corrects the certificate thumbprint.

azurecla commented:
Hi __@stegag__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stegag). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

PRmerger commented:
@stegag : Thanks for your contribution to the Azure documentation! The author, @yushwang, has been notified to review your proposed change.

stegag commented:
this change is not to be applied as it is wrong. sorry.

valdearg committed:
Correct Ubuntu distributions and signature

Corrects the error with the Ubuntu distribution names being incorrect. 
Simplifies the command to add repository. 
Corrects the certificate.
